trestle/core/control_interface.py

```diff
@@ -367,6 +392,63 @@ def get_param_vals_from_control_imp(control_imp: comp.ControlImplementation) ->
                 param_dict[set_param.param_id] = value_str
         return param_dict
 
+    @staticmethod
+    def merge_props(dest: Optional[List[common.Property]],
+                    src: Optional[List[common.Property]]) -> List[common.Property]:
+        """Merge a source list of properties into a destination list."""
+        if not src:
+            return dest
+        new_props: List[common.Propert] = []
```

Comment:
common.Property

Reply:
As discussed, the part_a. thing is working as designed.  Normally a part like that would not be given a name ending with dot.  There is a special case for labels such as a. when they correspond to a known label of a statement part - in which case no double dots will appear.  This is one of the existing test cases.

I fixed the Propert typo.